src/Plugin/Action/FetchEntityById.php

```diff
@@ -87,9 +88,9 @@ public function summary() {
    * {@inheritdoc}
    */
   public function execute() {
-    $entity_type = $this->getContextValue('entity_type');
+    $entity_type_id = $this->getContextValue('entity_type_id');
     $entity_id = $this->getContextValue('entity_id');
-    $storage = $this->entityManager->getStorage($entity_type);
+    $storage = $this->entityManager->getStorage($entity_type_id);
     $entity = $storage->load($entity_id);
     $this->setProvidedValue('entity', $entity);
```

Comment:
We should have a todo here for the refinement of the provided context definition for 'entity'. Example: if the loaded entity is a node then the provided context definition should use the type node. We don't have an API for that yet, though.